### DISCUSSION TITLE
skywalking-apm-8.7.0  protobuf-maven-plugin  compile  failed

### DISCUSSION THREAD

**Discussion by freeport0421** [apache/skywalking#10789] [General]
Request for help

[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: windows
[INFO] os.detected.arch: x86_64
[INFO] os.detected.version: 10.0
[INFO] os.detected.version.major: 10
[INFO] os.detected.version.minor: 0
[INFO] os.detected.classifier: windows-x86_64
[INFO]
[INFO] --- protobuf-maven-plugin:0.6.1:compile (grpc-build) @ apm-network ---
[INFO] Compiling 20 proto file(s) to D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\java
[ERROR] PROTOC FAILED: management/Management.proto:44:29: Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\browser\BrowserPerf.proto [0:0]: management/Management.proto:44:29: Expected type name.

proto文件编译报错，求助各位大佬怎么解决啊

**Comment by freeport0421** [comment]
[INFO] --- os-maven-plugin:1.6.2:detect (default) @ apm-network ---
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: windows
[INFO] os.detected.arch: x86_64
[INFO] os.detected.version: 10.0
[INFO] os.detected.version.major: 10
[INFO] os.detected.version.minor: 0
[INFO] os.detected.classifier: windows-x86_64
[INFO]
[INFO] --- protobuf-maven-plugin:0.6.1:compile (grpc-build) @ apm-network ---
[INFO] Compiling 20 proto file(s) to D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\java
[ERROR] PROTOC FAILED: management/Management.proto:44:29: Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\browser\BrowserPerf.proto [0:0]: management/Management.proto:44:29: Expected
type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\browser\BrowserPerfCompat.proto [0:0]: management/Management.proto:44:29: Exp
ected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\common\Common.proto [0:0]: management/Management.proto:44:29: Expected type n
ame.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\event\Event.proto [0:0]: management/Management.proto:44:29: Expected type nam
e.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\CLRMetric.proto [0:0]: management/Management.proto:44:29: Expe
cted type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\CLRMetricCompat.proto [0:0]: management/Management.proto:44:29
: Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\ConfigurationDiscoveryService.proto [0:0]: management/Manageme
nt.proto:44:29: Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\JVMMetric.proto [0:0]: management/Management.proto:44:29: Expe
cted type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\JVMMetricCompat.proto [0:0]: management/Management.proto:44:29
: Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\Meter.proto [0:0]: management/Management.proto:44:29: Expected
 type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\MeterCompat.proto [0:0]: management/Management.proto:44:29: Ex
pected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\Tracing.proto [0:0]: management/Management.proto:44:29: Expect
ed type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\language-agent\TracingCompat.proto [0:0]: management/Management.proto:44:29:
Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\logging\Logging.proto [0:0]: management/Management.proto:44:29: Expected type
 name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\management\Management.proto [0:0]: management/Management.proto:44:29: Expecte
d type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\management\ManagementCompat.proto [0:0]: management/Management.proto:44:29: E
xpected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\profile\Profile.proto [0:0]: management/Management.proto:44:29: Expected type
 name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\profile\ProfileCompat.proto [0:0]: management/Management.proto:44:29: Expecte
d type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\service-mesh-probe\service-mesh-compat.proto [0:0]: management/Management.pro
to:44:29: Expected type name.

[ERROR] D:\github\apache-skywalking-apm-8.7.0\apm-protocol\apm-network\src\main\proto\service-mesh-probe\service-mesh.proto [0:0]: management/Management.proto:44:2
9: Expected type name.

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for apm-protocol 8.7.0:
[INFO]
[INFO] apm-protocol ....................................... SUCCESS [  0.855 s]
[INFO] apm-network ........................................ FAILURE [  0.932 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.910 s
[INFO] Finished at: 2023-05-11T10:55:29+08:00
[INFO] ------------------------------------------------------------------------
[WARNING] The requested profile "dev" could not be activated because it does not exist.
[ERROR] Failed to execute goal org.xolstice.maven.plugins:protobuf-maven-plugin:0.6.1:compile (grpc-build) on project apm-network: protoc did not exit cleanly. Rev
iew output for more information. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :apm-network

**Comment by wu-sheng** [comment]
8.x series is EOL.
